src/storage/src/hummock/mod.rs

```diff
@@ -271,6 +274,18 @@ impl HummockStorage {
     pub fn get_pinned_version(&self) -> PinnedVersion {
         self.pinned_version.load().deref().deref().clone()
     }
+
+    pub fn need_write_throttling(&self) -> bool {
```

Comment:
Do we need to distinguish between different groups of l0 ? , I think their write paths are independent